app/src/components/map/DatasetPopup.tsx

```diff
@@ -0,0 +1,124 @@
+import Box from '@material-ui/core/Box';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Collapse from '@material-ui/core/Collapse';
+import Link from '@material-ui/core/Link';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import Typography from '@material-ui/core/Typography';
+import { SPATIAL_COMPONENT_TYPE } from 'constants/spatial';
+import { Feature } from 'geojson';
+import { useApi } from 'hooks/useApi';
+import useDataLoader from 'hooks/useDataLoader';
+import React from 'react';
+import { useHistory } from 'react-router';
+
+export type BoundaryCentroidFeature = Feature & { properties: BoundaryCentroidFeatureProperties };
+
+export type BoundaryCentroidFeatureProperties = {
+  type: SPATIAL_COMPONENT_TYPE.BOUNDARY_CENTROID;
+};
+
+interface IDatasetSpatialMetadata {
```

Comment:
Can you add these values to the `BoundaryCentroidFeatureProperties` above, and use that instead below.